### DISCUSSION TITLE
Next big release features

### DISCUSSION THREAD

**Discussion by justquick** [justquick/django-activity-stream#513] [Ideas]
## 👋 Hi All
  Im looking to discuss the upcoming new features for django-activity-stream. There are some big features in the pipe but I need some help formulating the priorities and milestones. The next big features are 
  * Finally DRF! This work is in a draft and is almost ready to go. Please review!: #502 
  * The next AS2 spec. https://www.w3.org/TR/activitystreams-core/ This is probably the biggest modernization this app could use but has a few moving parts
  * * #500 
  * * #501 
  * * #389 
  
And any new Ideas. All are welcome to contribute to the direction of this app. To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by auvipy** [comment]
that's great
